oap-ape/ape-native/src/utils/GroupByUtils.h

```diff
@@ -34,7 +34,7 @@ class GroupByUtils {
                       const std::vector<int64_t>& nullPtr, std::vector<Key>& keys,
```

Comment:
Grouped keys may also need check if column values are `null`.
`null` is allowed in group keys, but it will require `PartialKey` and `ApeHashMap` to be updated for hash and equality checking.

Reply:
Good catch! We can have more discussion about how to handle such case, and I will fix it in further PRs.